runtime/tests/automl_runtime/forecast/utils_test.py

```diff
@@ -0,0 +1,72 @@
+#
+# Copyright (C) 2022 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+
+import pandas as pd
+
+from databricks.automl_runtime.forecast.utils import generate_cutoffs
+
+
+class TestGenerateCutoffs(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.X = pd.DataFrame(
+            pd.date_range(start="2020-07-01", end="2020-08-30", freq='d'), columns=["ds"]
+        ).rename_axis("y").reset_index()
+
+    def test_generate_cutoffs_success(self):
```

Comment:
Do we also want to test the failed conditions for `generate_cutoffs`?

Reply:
We have the two failure cases below - `test_generate_cutoffs_failure_horizon_too_large` and `test_generate_cutoffs_less_data`. Is there other failure cases to cover?